src/jvmTest/kotlin/logcat/LogcatTest.kt

```diff
@@ -0,0 +1,167 @@
+package logcat
+
+import logcat.LogPriority.INFO
+import kotlin.test.AfterTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class LogcatTest {
```

Comment:
Why is this test duplicated in 2 places?

Reply:
As much as possible, commonTest includes test that can run on all platforms.  They are executed once for each of the platforms (JVM/JS/Native) using the same test.